### ISSUE TITLE
bundle-sfx broken ("SFX exclusion ... needs a reference.")

### ISSUE THREAD

**Issue by neothemachine** [jspm/jspm-cli#1644]
I just updated from jspm 0.16.14 (or something in that area) to 0.16.31 and now get this error:

```
> jspm bundle-sfx leaflet-coverage/classic - leaflet - c3 leaflet-coverage.src.js --globals "{leaflet:'L', c3:'c3'}" --format global --skip-source-maps

     Building the single-file sfx bundle for leaflet-coverage/classic - leaflet - c3...

err  SFX exclusion "github:Leaflet/Leaflet@0.7.7.js" needs a reference.
     Either output an SFX module format like --format amd or map the module to an environment global via --globals "{'test.js': 'test'}".
```

The package.json has `"leaflet": "github:Leaflet/Leaflet@^0.7.3",` under jspm dependencies.

You can reproduce this by cloning https://github.com/Reading-eScience-Centre/leaflet-coverage and running `npm install`.

**Comment by guybedford** [comment]
Thanks this sounds like a regression. `defaultJSExtensions` is always a pain! Will look into it when I can.

**Comment by guybedford** [comment]
This actually turned out to be an obscure casing bug. Thanks for the test case, fixed in https://github.com/systemjs/builder/commit/72cb94aa6cbdc3dfaf697ccfb1066b5c162aa165.

**Comment by guybedford** [comment]
Fixed in jspm 0.17.0-beta.12.

**Comment by neothemachine** [comment]
Thanks, works!
